skimage/morphology/convex_hull.py

```diff
@@ -47,7 +48,11 @@ def convex_hull_image(image, offset_coordinates=True, tolerance=1e-10):
 
     """
     ndim = image.ndim
-
+    if not np.max(image):
```

Comment:
Could you add a test case covering this change?

Reply:
done

Reply:
Hmm... On a second though this could work wrong with intXX or floatXX images with non-positive values. Should we have a stricter check here? Otherwise, LGTM.

/offtopic LOL, I've just noticed that we have a series of lies in the docstrings: https://github.com/scikit-image/scikit-image/blob/master/skimage/morphology/convex_hull.py#L29 (actually, isn't cast);  https://github.com/scikit-image/scikit-image/blob/master/skimage/morphology/_convex_hull.pyx#L10, https://github.com/scikit-image/scikit-image/blob/master/skimage/morphology/_convex_hull.pyx#L15 (not boolean).